src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -200,7 +200,8 @@ public static BaselineStateVersionOne CreateBaselineStateVersionOne(BaselineStat
             {
                 case BaselineState.Absent:
                     return BaselineStateVersionOne.Absent;
-                case BaselineState.Existing:
+                case BaselineState.Unchanged:
+                case BaselineState.Updated:
```

Comment:
So we lose information on the round trip (v2 Updated => v1 Existing => v2 Unchanged), but oh well.

Reply:
That's true. We could persist something in a property bag, of course. But we don't foresee need for detailed round-tripping of this level of precision currently. Can revisit if need be (don't think either of us would put odds on it).

---
In reply to: [251997378](https://github.com/Microsoft/sarif-sdk/pull/1241#discussion_r251997378) [](ancestors = 251997378)